sdk/cosmosdb/cosmos/test/internal/unit/sasToken.spec.ts

```diff
@@ -1,12 +1,10 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 import assert from "assert";
-import { CosmosClient } from "../../../dist-esm/";
+import { CosmosClient } from "@azure/cosmos";
 import { endpoint } from "../../public/common/_testConfig";
 import { masterKey, userSasTokenKey } from "../../public/common/_fakeTestSecrets";
-import { SasTokenPermissionKind } from "../../../dist-esm/common";
-import { createAuthorizationSasToken } from "../../../dist-esm/utils/SasToken";
-import { SasTokenProperties } from "../../../dist-esm/client/SasToken/SasTokenProperties";
+import { SasTokenProperties,SasTokenPermissionKind,createAuthorizationSasToken } from "@azure/cosmos";
```

Comment:
since all the imports are from @azure/cosmos and so is the CosmosClient then why not have them in the same import statement?

Reply:
I recommend running the "organize imports" source action in VSCode. It will combine and sort imports.

Reply:
since these are tests , i have changed it to import from the src